app/helpers/application_helper.rb

```diff
@@ -26,7 +26,8 @@ def govuk_back_link_to(url = :back, body = "Back")
     )
   end
 
-  def link_to_runner(runner_url, form_id)
-    "#{runner_url}/preview-form/#{form_id}"
+  def link_to_runner(runner_url, form_id, live = false)
+    mode_segment = live ? "form" : "preview-form"
+    "#{runner_url}/#{mode_segment}/#{form_id}"
   end
```

Comment:
I was thinking maybe pass in the form and let the helper get access to the info it needs to work out what link to use
```suggestion
  def link_to_runner(runner_url, form)
    mode_segment = "preview-" if form.live?
    "#{runner_url}/#{mode_segment}form/#{form.id}"
  end
```

Reply:
🤦  sorry meant to use the invert of `.live?`

Reply:
Yeah, that could work and it's quite neat. I suppose what we do depends on what we expect the various 'Preview this form' links to do for a live form - should they go to the live form URL or to the preview version?

Reply:
@aliuk2012 to be clearer on that - I think if we do it by inverting .live, then all of the 'Preview this form' links will point to the live URL rather than the preview. whereas if we pass a 'live' boolean to the helper we can pick and choose which ones point to live routes and which point to the preview links. I'm not actually sure which of those we want here